telegram/ext/basepersistence.py

```diff
@@ -183,12 +194,23 @@ def insert_bot(self, obj: object) -> object:  # pylint: disable=R0911
         """
         if isinstance(obj, Bot):
             return self.bot
-        if obj == self.REPLACED_BOT:
+        if isinstance(obj, str) and obj == self.REPLACED_BOT:
             return self.bot
         if isinstance(obj, (list, tuple, set, frozenset)):
             return obj.__class__(self.insert_bot(item) for item in obj)
 
-        new_obj = copy(obj)
+        try:
+            new_obj = copy(obj)
+        except Exception:
+            warnings.warn(
+                'BasePersistence.insert_bot caught an error while trying to copy an object. '
+                'Objects that can not be copied will be assumed to not contain a telegram.Bot '
+                'instance and will not be handled further. See the docs of '
+                'BasePersistence.insert_bot for more information.',
+                RuntimeWarning,
+            )
```

Comment:
Try to make this text shorter.
People will get bored and look away with such a long text...